pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md

```diff
@@ -37,10 +37,11 @@ This API supports OAuth 1.0 authentication patterns and requires SSL protocol. P
 | v | API version  | string | Required |
 | sid | Session ID | string | Required |
 
-<div class="attn-note">
```

Comment:
As earlier, this is not several notes, but one. Make one paragraph if need be for the moment, but change again once multi-paragraph HTML markup for notes is supported.